client/css/style.css

```
@@ -0,0 +1,235 @@
+/* End header styles */
+:root {
+    --orange: rgb(255,110,74);
+    --orange-dark: rgb(242,92,55);
+
+    --teal: rgb(27,235,185);
+    --teal-dark: rgb(12,226,176);
+    --teal-very-dark: rgb(4, 126, 97);
+
+    --blue: rgb(142, 219, 255);
+
+    --dark: rgb(22, 22, 24);
+    --dark-muted: rgb(86, 86, 92);
+
+    --light: rgb(255,255,255);
+    --light-slightly-muted: rgb(234, 234, 234);
+    --light-muted: rgb(145, 145, 145);
+
+}
+body {
+    font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+}
+
+h2 {
+    font-size: 1.25rem;
+}
+
+#container {
+    margin-top: 52px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#container a {
+    color: var(--orange-dark);
+    text-decoration: none;
+}
+
+#container a:hover:enabled {
+    text-decoration: underline;
+}
+
+#controls {
+    display: flex;
+    flex-direction: row;
+    min-height: 50vh;
+}
+
+#vectorStore {
+    flex: 1 1 140px;
+    border: var(--dark-muted) 1px solid;
+    border-radius: 8px;
+    padding: 12px;
+}
+
+#vectorStore #status {
+    display: flex;
+    border-radius: 4px;
+    background-color: var(--light-slightly-muted);
+    padding: 4px;
+}
+
+#vectorStore #status #state {
+    display: flex;
+    color: var(--orange);
+    font-weight: bold;
+}
+
+#pendingIndexing {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+#uploadControls {
+    border: var(--light-muted) 1px solid;
+    padding: 8px;
+    border-radius: 4px;
+    background-color: var(--light-slightly-muted);
+}
+
+#query {
+    display: flex;
+    flex: 1 1 auto;
+    flex-direction: column;
+    justify-content: flex-start;
+    border: var(--dark-muted) 1px solid;
+    margin-left: 8px;
+    border-radius: 8px;
+    padding: 12px;
+}
+
+#response {
+    margin: 12px 0px;
+}
+
+input[type="text"],textarea {
+    border-radius: 4px;
+    font-size: 1.2em;
+    width: 100%;
+}
+
+textarea {
+    height: 70px;
+}
+#recordings {
+    display: flex;
+    flex-direction: column;
+}
+
+#manual {
+    border: var(--dark-muted) 1px solid;
+    width: 100%;
+    border-radius: 8px;
+    padding: 12px;
+}
+
+#ask {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+#uploadError {
+    color: red;
+    font-size: 0.8em;
+}
+
+
+.lobby {
+    display: flex;
+    flex-direction: column;
+    width: 70%
+}
+
+.lobby a {
+    color: var(--orange-dark);
+    text-decoration: none;
+}
+
+.lobby a:hover:enabled {
+    text-decoration: underline;
+}
+
+
+
+/* Form and button styling */
+
+form {
+    margin-bottom: 8px;
+    margin-bottom: 8px;
+    display: flex;
+    flex-direction: column;
+}
+
+form button {
+    font-size: 1.2em;
+    margin: 4px;
```

```suggestion
    margin: 8px 0;
```
Nit: not every button has a horizontal margin so I would just remove it for consistency.